use_case_examples/lora_finetuning/requirements.txt

```diff
@@ -4,6 +4,6 @@ peft==0.12.0
 Jinja2==3.1.4
 matplotlib==3.7.5
 datasets==3.1.0
-accelerate==1.2.0
+accelerate==1.0.1
```

Comment:
Which package the 1.2.0 version had conflicts with?

Reply:
1.2.0 isn't compatible with python 3.8. I will probably revert this and add a comment that we don't support 3.8 in this use case

Reply:
yes please add a comment